Resources/Prototypes/Maps/gaxstation.yml

```
@@ -0,0 +1,70 @@
+- type: gameMap
+  id: Gax
+  mapName: 'NCS Gax'
+  mapPath: /Maps/gaxstation.yml
+  minPlayers: 10
+  maxPlayers: 45
+  fallback: true
+  stations:
+    Gaxstation:
+      stationProto: StandardNanotrasenStation
+      components:
+        - type: StationEmergencyShuttle
+          emergencyShuttlePath: /Maps/Shuttles/DeltaV/NTES_Delta.yml
+        - type: StationNameSetup
+          mapNameTemplate: '{0} NCS Gax {1}'
+          nameGenerator:
+            !type:NanotrasenNameGenerator
+            prefixCreator: '14'
+        - type: StationJobs
+          overflowJobs:
+          - Passenger
+          availableJobs:
+            #service
+            Captain: [ 1, 1 ]
+            HeadOfPersonnel: [ 1, 1 ]
+            Bartender: [ 1, 1 ]
+            Botanist: [3 , 3 ]
+            Chef: [ 2, 2 ]
+            Janitor: [ 2, 2 ]
+            ServiceWorker: [ 2, 2 ]
+            #Engineering
+            ChiefEngineer: [ 1, 1 ]
+            AtmosphericTechnician: [ 4, 4 ]
+            StationEngineer: [ 4, 4 ]
+            TechnicalAssistant: [ 2, 2 ]
+            #Medical
+            ChiefMedicalOfficer: [ 1, 1 ]
+            Chemist: [ 2, 2 ]
+            MedicalDoctor: [ 3, 3 ]
+            MedicalIntern: [ 3, 3 ]
+            Paramedic: [ 2, 2 ]
+            MedicalBorg: [ 1, 1 ]
+            #Epistemics
+            ResearchDirector: [ 1, 1 ]
+            Scientist: [ 4, 4 ]
+            ResearchAssistant: [ 2, 2 ]
+            Roboticist: [ 1, 2 ]
+            Borg: [ 2, 2 ]
+            Chaplain: [ 1, 1 ]
+            Librarian: [ 1, 1 ]
+            #Security
```

```suggestion
            Chaplain: [ 1, 1 ]
            Librarian: [ 1, 1 ]
            ForensicMantis: [ 1, 1 ]
            #Security
```
Will also need a spawner added.